bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -5430,15 +5460,15 @@ msgstr ""
 #: bika/lims/browser/dashboard/dashboard.py:727
 #: bika/lims/profiles/default/workflows/bika_sample_workflow/definition.xml
 msgid "Sample received"
-msgstr ""
+msgstr "Muestra recepcionada"
```

Comment:
@xispa I have doubts whether _recepcionada_ is better than _recibida_.

 _recibida_ sounds more natural to me, but maybe the technical word is _recepcionada_ so I am ok with whatever you prefer.